ci-operator/config/opendatahub-io/odh-dashboard/opendatahub-io-odh-dashboard-main.yaml

```diff
@@ -39,6 +39,13 @@ tests:
     env:
       IMAGE_REPO: odh-dashboard
     workflow: opendatahub-io-ci-image-mirror
+- as: odh-dashboard-s390x-build
+  cron: 0 2 * * *
+  steps:
+    env:
```

Comment:
One quick question, since we are not using the `SOURCE_IMAGE_REF` does it make sense to place this here rather than in another file?
I feel like we don't wanna use the promotion neither the image.
(Not sure, Im just asking I'm not that well versed in openshift CI)

Reply:
Yes, you're correct. We are not using promotion because we don't have s390x OCP cluster profile yet and so the  SOURCE_IMAGE_REF is not needed.

Reply:
Ok great!